gettsim/interface.py

```diff
@@ -9,11 +8,13 @@
 from gettsim.config import PATHS_TO_INTERNAL_FUNCTIONS
 from gettsim.dag import _dict_subset
 from gettsim.dag import create_dag
-from gettsim.dag import create_function_dict
 from gettsim.dag import execute_dag
-from gettsim.dag import prune_dag
+from gettsim.dag import partial_parameters_to_functions
+from gettsim.dag import remove_parameter_nodes
 from gettsim.functions_loader import convert_paths_to_import_strings
 from gettsim.functions_loader import load_functions
+from gettsim.shared import format_list_linewise
+from gettsim.shared import parse_to_list_of_strings
 
 
 def compute_taxes_and_transfers(
```

Comment:
`user_functions` should become `functions` as per comment above and [discussion on Zulip](https://gettsim.zulipchat.com/#narrow/stream/237340-DAGs/topic/user_functions.20vs.2E.20internal_functions/near/201864925)